tests/common/gu_utils.py

```diff
@@ -16,7 +16,7 @@
 
 BASE_DIR = os.path.dirname(os.path.realpath(__file__))
 FILES_DIR = os.path.join(BASE_DIR, "files")
-TEMPLATES_DIR = os.path.join(BASE_DIR, "templates")
+TEMPLATES_DIR = os.path.join(BASE_DIR, "../generic_config_updater/templates")
```

Comment:
There would still have dependency issue because the template is under generic_config_updater/, how about separate gu_utils.py into 2 files? File under common/ contains functions shared by all modules, file under generic_config_updater/ contains functions and templates using by gcu module

Reply:
I think this is not a good idea, which will cause many duplicate code.

Reply:
I can also refactor this templates file

Reply:
I can also refactor these templates

Reply:
It won't cause duplicate code, for example, you can put function `apply_patch` under common/, and leave function related to gcu, then they can import `apply_patch` from common/
For refactor templates, I think it doesn't make sense if you move templates to common/ but they were only used by gcu, @wangxin do you have any suggestion?

Reply:
Oh, I get what you mean, let me fix it.